docs/book/src/concepts.md

```diff
@@ -30,6 +30,7 @@ Currently supported providers:
 - [Azure Provider](https://azure.github.io/secrets-store-csi-driver-provider-azure/)
 - [GCP Provider](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp)
 - [Vault Provider](https://github.com/hashicorp/secrets-store-csi-driver-provider-vault)
+- [Akeyless Provider](https://github.com/akeylesslabs/akeyless-csi-provider)
```

Comment:
Let's make the doc updates in a separate PR after:

1. Tests are passing and merged
2. Code audit of the provider is complete

Reply:
removed